s2s_vpn_tunnel.tf

```diff
@@ -0,0 +1,59 @@
+# ------------------------------------------------------------------------------
+# Create a tunnel to VENOM
+# ------------------------------------------------------------------------------
+
+resource "aws_customer_gateway" "venom" {
+  provider = aws.sharedservicesprovisionaccount
+
+  bgp_asn    = 65000 # Unused
+  ip_address = "52.177.137.15"
```

Comment:
Should this be a Terraform variable?

Reply:
Well this code is VENOM-specific, and that IP address is _the_ IP for VENOM's end of the tunnel.  If I were going to make it a variable then I should make all the other tunnel parameters variables as well, although I don't think that makes this code any more reusable.

Reply:
I was thinking that the VENOM folks would probably prefer that their IP address was not included in our code.  Also, if it ever changes, it would not require a code change on our end (kind of a weak argument I know, but it's true).

I understand that this isn't consistent with the other tunnel parameter variables (which I don't think need to be TF variables), but IP addresses can fall into the category of "sensitive data".  Not saying we are hiding it, but no need to advertise it more than necessary.

Anyway, I'm not going to die on this hill if you feel strongly about this.